src/mon/OSDMonitor.cc

```diff
@@ -3346,7 +3346,8 @@ void OSDMonitor::tick()
 	}
 
 	if (g_conf->mon_osd_down_out_interval > 0 &&
-	    down.sec() >= grace) {
+	    down.sec() >= grace &&
+            !osdmap.is_destroyed(o)) {
```

Comment:
Yeah, that makes sense.  But perhaps we make it explicit by adding a mon_osd_destroyed_out_interval option, with behaiior identical do the down option?